assets/src/edit-story/components/tablist/styles.js

```diff
@@ -93,9 +93,9 @@ export const PanelWrapper = styled.div`
   width: 100%;
   background: ${({ theme }) => theme.colors.bg.primary};
 
-  :not(:last-child) ${TabButton} {
-    box-shadow: 0px 1px 0 0 ${({ theme }) => theme.colors.divider.tertiary};
-    margin-bottom: 1px;
+  &:not(:first-child) > ${TabButton} {
+    box-shadow: 0px -1px 0 0 ${({ theme }) => theme.colors.divider.tertiary};
+    margin-top: 1px;
```

Comment:
fixes shadow styles when buttons are missing

Reply:
I did a small change here too. might cause some conflicts, but just gotta keep track of which merges first and update the other:
https://github.com/google/web-stories-wp/pull/8225